Provisioning swagger document

mvahlsing98117 opened the pull request:
File name change
File has been updated and tested.

mvahlsing98117 committed:
Add files via upload

I am receiving a "Fail to fetch" error when testing my file. I believe that its' a server configuration error that does not allow me to test across domains. 
I want to test this on the developer portal, please do release this as the final version. 


Failed to load resource: net::ERR_NETWORK_CHANGED
(index):1 Uncaught (in promise) TypeError: Failed to fetch
index.js:581 Mixed Content: The page at 'https://editor.swagger.io/' was loaded over HTTPS, but requested an insecure resource 'http://us.api.concursolutions.com/api/provisioning/v1/provisions'. This request has been blocked; the content must be served over HTTPS.
(anonymous) @ index.js:581
swagger-ui.js:1 TypeError: Failed to fetch
(anonymous) @ swagger-ui.js:1
(index):1 Access to fetch at 'https://us.api.concursolutions.com/api/provisioning/v1/provisions' from origin 'https://editor.swagger.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The 'Access-Control-Allow-Origin' header contains the invalid value ''. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

mvahlsing98117 committed:
Rename UserProvisioning_Testing1.json to provisioning_swagger.json.json

File name change

mvahlsing98117 committed:
Rename provisioning_swagger.json.json to provisioning_swagger.json

Fixing file name

MarcBell-SAP commented on swagger/draft/provisioning_swagger.json:
```diff
@@ -0,0 +1,619 @@
+        }
+      }
+    },
+    "OperationsStatusResonse": {
+      "properties": {
+        "completed": {
+          "type": "string"
+        },
+        "extentions": {
+          "$ref": "StatusResponse"
+        }
+      }
+    },
```
This object is never used. Please remove this from the document.

MarcBell-SAP commented on swagger/draft/provisioning_swagger.json:
```diff
@@ -0,0 +1,619 @@
+    },
+    "ExtensionResponse": {
+      "properties": {
+        "message": {
+          "type": "array"
+        },
```
This object requires an items property because it's an array. See https://swagger.io/docs/specification/data-models/data-types/#array for reference.

mvahlsing98117 committed:
Update provisioning_swagger.json

Removed OperationsStatusResonse based on suggestion from Marc Bell
Modified ExtensionResponse to reply with an Object vs. array.

jonikamelcher commented:
:shipit:
